core/templates/core/panel.html

```diff
@@ -0,0 +1,53 @@
+{% extends 'core/base.html' %}
+
+{% block title %}Panel Gwardii{% endblock %}
+
+{% load list_events %}
+
+{% block content %}
+    <div class="flex flex-row nowrap flex-grow h-full overflow-auto">
+        <div id="sidebar" class="flex md:flex flex-col gap-y-3 fixed md:relative top-0 left-[-18rem] md:left-0 md:translate-x-0 min-w-72 max-w-72 h-full bg-gray-800 shadow-2xl overflow-auto duration-300 md:duration-0 border-r border-gray-600">
```

Comment:
Why to set min-w and max-w when you could just set `w`?

Reply:
I've already told you that for some reason I have to set min-w